recipe-tests.yaml

```diff
@@ -10,4 +19,6 @@
       - krb5-devel
       - rpm-libs
       - redhat-rpm-config
+      - gcc
+      - rpm-devel
```

Comment:
I'm not sure about `rpm-devel`. It works for me locally and in [CI](https://github.com/packit-service/packit/blob/master/Jenkinsfile#L34) without it.

Reply:
comment it out, run `make build-tests && make test-in-container` and see it yourself: it failed for me and the message literally was: `I need rpm-devel`.

Reply:
Ok, keep it there.

> make build-tests

I can't because of https://github.com/ansible-community/ansible-bender/issues/101

> make test-in-container

I can't because you commented it out

Reply:
I asked if you are still hitting the issue and you did not respond.

Okay, since this PR doesn't really make the tests run in a container, I will comment out more stuff and just mention current status.

Reply:
Didn't I? https://github.com/packit-service/packit/pull/215#issuecomment-479461874